wazo_calld/plugins/voicemails/storage.py

```diff
@@ -243,6 +246,10 @@ def _parse_callerid(value, result):
         elif caller_id.is_complete_caller_id(value):
             result['caller_id_name'] = caller_id.extract_displayname(value)
             result['caller_id_num'] = caller_id.extract_number(value)
+        elif value != value.lstrip() and caller_id.is_complete_caller_id(value.lstrip()):
```

Comment:
I think this `elif` is not required anymore since all lines are cleaned at the top of the `for` loop.